src/libraries/System.IO.FileSystem.AccessControl/tests/FileSystemSecurityTests.cs

```diff
@@ -409,5 +416,85 @@ public void AuditRuleType_Returns_Valid_Object()
             Type type = fileSecurity.AuditRuleType;
             Assert.Equal(typeof(FileSystemAuditRule), type);
         }
+
+        [Fact]
+        public void DirNotFound_DirectorySecurity()
```

Comment:
This test and the next one verify that we throw from `NativeObjectSecurity` when the Windows error is `ERROR_PATH_NOT_FOUND`.